spec/unit/lib/temporal/workflow/execution_info_spec.rb

```diff
@@ -20,6 +20,13 @@
     it 'freezes the info' do
       expect(subject).to be_frozen
     end
+
+    it 'deserializes if search_attributes is nil' do
+      api_info.search_attributes = nil
+
+      result = described_class.generate_from(api_info)
+      expect(result.search_attributes).to be_nil
```

Comment:
Feels like this should be an empty hash and not `nil` (assuming there's no semantic difference). This would prevent SDK users from having to do their own nil-checks further down

Reply:
Fixed, let me know what you think!

Reply:
@antstorm I'm going to approve and merge since this comment seems to be resolved, but let me know if there were any other changes you wanted and I can open a fix PR.

Reply:
@antstorm I'm going to approve and merge since this comment seems to be resolved, but let me know if there were any other changes you wanted and I can open a fix PR.